package.json

```diff
@@ -1,51 +1,88 @@
 {
   "name": "@azure/amqp-common",
-  "version": "1.0.0-preview.1",
+  "version": "1.0.0-preview.2",
   "description": "Common library for amqp based azure sdks like @azure/event-hubs.",
   "author": "Microsoft Corporation",
   "license": "MIT",
-  "main": "./dist/lib/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/lib/index.js",
   "types": "./typings/lib/index.d.ts",
   "dependencies": {
+    "@azure/ms-rest-nodeauth": "^0.9.1",
+    "@types/is-buffer": "^2.0.0",
     "async-lock": "^1.1.3",
     "debug": "^3.1.0",
     "is-buffer": "^2.0.3",
     "jssha": "^2.3.1",
-    "ms-rest-azure": "^2.5.9",
-    "tslib": "^1.9.3"
+    "rhea": "^0.3.9",
+    "tslib": "^1.9.3",
+    "url": "^0.11.0",
+    "util": "^0.11.1"
```

Comment:
Neither `url` nor `util` packages are being used anywhere... Why are they in the dependencies?
Same for `tslib`

Reply:
url and util are both used by rhea and so need to be present here. I really wish I could add comments to package json files for this reason.

tslib is used by typescript when the importHelpers option is specified, so should be there as well I think!